File: .sdk/tm/ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { BluefinDecryptxP2peSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await BluefinDecryptxP2peSDK.test()
    equal(null !== testsdk, true)
  })

})

File: .sdk/tm/csharp/test/ExistsTest.cs
// BluefinDecryptxP2pe SDK exists test.

using Xunit;

using BluefinDecryptxP2peSdk;

namespace BluefinDecryptxP2peSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinDecryptxP2peSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}
